pkg/crc/api/handlers.go

```diff
@@ -214,9 +214,12 @@ func (h *Handler) UnsetConfig(c *context) error {
 }
 
 func (h *Handler) GetConfig(c *context) error {
+	queries := c.url.Query()
 	var req client.GetOrUnsetConfigRequest
-	if err := c.Bind(&req); err != nil {
-		return err
+	if len(queries) != 0 {
+		for key := range queries {
+			req.Properties = append(req.Properties, key)
```

Comment:
I don't know if golang calls https://pkg.go.dev/net/url#QueryUnescape automatically?

Reply:
@cfergeau Here we are using `url.Query` property which should handle it https://pkg.go.dev/net/url#URL.Query or you have any other specific suggestion?

Reply:
This was the question, whether it does it automatically or not. Seems to be fine https://cs.opensource.google/go/go/+/refs/tags/go1.16.7:src/net/url/url.go;l=927-959